openedx/adg/lms/webinars/views.py

```diff
@@ -50,6 +56,72 @@ def get_context_data(self, **kwargs):
             'registration': registration,
             'is_registered': is_registered,
             'send_post_call_to_register': send_post_call_to_register,
-            'featured_speakers': self.object.featured_speakers
+            'featured_speakers': self.object.featured_speakers,
+            'google_url': self.get_google_calendar_url(),
+            'yahoo_url': self.get_yahoo_calendar_url(),
         })
         return context
+
+    def get_google_calendar_url(self):
+        """
+        Generate url for the webinar to add the reminder on the google calendar
+
+        Returns:
+            str: google link
+        """
+        base_url = 'https://www.google.com/calendar/event?'
+        end_time = self.object.start_time + self.object.duration
+        details = {
+            'action': 'TEMPLATE',
+            'text': self.object.title,
+            'details': self.object.description,
+            'dates': self.object.start_time.strftime('%Y%m%dT%H%M%S') + '/' + end_time.strftime('%Y%m%dT%H%M%S'),
```

Comment:
Let's use a constant for `%Y%m%dT%H%M%S`

Reply:
We are not using this format again anywhere else.

Reply:
It's being used twice here. We can use a local constant